_posts/2018-01-22-LGBTSteminar2018.md

```diff
@@ -0,0 +1,23 @@
+---
+layout:     post
+title:      LGBT STEMinar
+date:       2018-01-22 09:00
+summary:    Our lessons from the 2018 LGBT STEMinar
+category:   Conferences
+author:     alice_kaerast
+---
+
+Sky Betting & Gaming send people to a large number of different conferences each year because this is a great way to learn from our peers in other organisations.  It is incredibly valuable to see how other companies approach similar issues, especially in areas of cutting edge technology where more traditional training courses often lag behind.
+
+Whilst many of these conferences are directly-related to our day-jobs, we also attend conferences that are only tangentially related.  We have an awful lot to learn from tangential fields to our own, and so one event I recently attended was the annual LGBT Steminar hosted at the University of York’s STEM Learning Centre.  I spent the day learning about some of the latest scientific research that the LGBT+ people presenting have been involved in, as well as learning a lot about the state of LGBT representation in academia.
```

Comment:
The event itself is "LGBT Steminar", so that's fine. But everywhere else, we should be consistent with whether we use "LGBT" or "LGBT+" (the latter is better)

Reply:
I've caught the last few cases of "LGBT" and changed to LGBT+